### DISCUSSION TITLE
Proxmox Backup Server LXC / NFS Datastore / shutdown problem

### DISCUSSION THREAD

**Discussion by ts-ws** [tteck/Proxmox#3475] [Help Needed]
Hi,
I run two PBS in LXC both show the same behavoir. I did the same before with Full VM's where I didn't encounter the Problem.
I mount a NFS share to my PBS over fstab which I use to store the "PBS Datastore" so the backups saved on my NAS.
`NAS-FQDN:/volume1/backup_proxmox/pbs-p01 /mnt/hssynonode-p01 nfs nfsvers=4,minorversion=1,rw,_netdev,nofail 0 0`
This is working very well. The problem is if I try to shutdown (or restart) the LXC it takes between 5 to 15 min to do the shutdown. If I remove the NFS mount it only takes secounds.
During the shutdown there is no aktive job running.
I think the system is waiting for some kind of timeout related to NFS, but I can't figure out where it stucks and what I can do against it.

**Comment by sugar012** [comment]
sorry to hijack your question. did you set the LXC privileged? I set it uprivileged and not able to mount

**Reply by ts-ws** [reply]
Hi,
these are my settings
![image](https://github.com/user-attachments/assets/d8c8699f-956f-4755-b9f1-03b0d2556d0e)

**Comment by Toastyyy3** [comment]
AFAIK NFS freezes the system trying to access it once NFS is unreachable. So idk if that pushes you towards the right direction but you could look into systemd automount function. I believe there is an option to auto-unmount an nfs drive when it's not used for a certain time.
@sugar012 I believe you can never mount a drive inside an unprivileged container. It has to be either privileged or you have to mount it on the host and mount bind it in the config file of the unprivileged container.

**Comment by pax0707** [comment]
I am encountering the same issue here.
